commit 72547f774641d3ed881cc957a6d6c20d5e3370ec
Author: Louis Sautier <sautier.louis@gmail.com>
Date:   Mon Apr 20 16:40:08 2020 +0200

    Fix build with musl by including limits.h when PATH_MAX is used
    
    Otherwise, the build fails with:
    error: PATH_MAX undeclared (first use in this function)
    
    Bug: https://bugs.gentoo.org/717038

diff --git a/lib/processes.c b/lib/processes.c
index cb006ca..e65992d 100644
--- a/lib/processes.c
+++ b/lib/processes.c
@@ -26,6 +26,7 @@
 #include <ctype.h>
 #include <dirent.h>
 #include <errno.h>
+#include <limits.h>
 #include <pwd.h>
 #include <stdio.h>
 #include <stdlib.h>
diff --git a/plugins/check_fc.c b/plugins/check_fc.c
index b798377..cfaafa5 100644
--- a/plugins/check_fc.c
+++ b/plugins/check_fc.c
@@ -22,6 +22,7 @@
 #include <dirent.h>
 #include <errno.h>
 #include <getopt.h>
+#include <limits.h>
 #include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
